operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -420,295 +476,406 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
-        } 
-            
+            addSearchParameterIfNotExcluded(compartmentMemberType, "prescriber", ResourceType.Value.PRACTITIONER, searchParameters, allowedIncludes, extraResources, supportedSearchParametersMap);
+        }
+
         // BodyStructure, ResearchSubject, and RelatedPerson have no references to:
         // Location, Medication, Organization, and Practitioner
-        
-        // DeviceUseStatement, EnrollmentRequest, FamilyMemberHistory, ChargeItem, 
-        // ImmunizationEvaluation, ImmunizationRecommendation, MolecularSequence, 
-        // and NutritionOrder have no search parameters that reference: 
+
+        // DeviceUseStatement, EnrollmentRequest, FamilyMemberHistory, ChargeItem,
+        // ImmunizationEvaluation, ImmunizationRecommendation, MolecularSequence,
+        // and NutritionOrder have no search parameters that reference:
         // Location, Medication, Organization, and Practitioner
     }
-    
-    private void addSearchParameterIfNotExcluded(String compartmentMemberType, String code, 
-        ResourceType.Value subResourceType, MultivaluedMap<String, String> searchParameters, 
-        List<String> allowedIncludes) {
+
+    private void addSearchParameterIfNotExcluded(String compartmentMemberType, String code,
+            ResourceType.Value subResourceType, MultivaluedMap<String, String> searchParameters,
+            List<String> allowedIncludes, List<String> extraResources, ParametersMap supportedSearchParametersMap) {
         // Need to make sure the search parameter has not been excluded
         String parameterName = compartmentMemberType + ":" + code + ":" + subResourceType.value();
         String simplifiedParameterName = compartmentMemberType + ":" + code;
-        if (allowedIncludes == null || allowedIncludes.contains(parameterName)
-                || allowedIncludes.contains(simplifiedParameterName)) {
+        // Also search if parameter name is supported by looking at the list of search
+        // parameters
+        SearchParameter searchParam = supportedSearchParametersMap.lookupByCode(code);
+        if ((extraResources != null && extraResources.contains(subResourceType.value())) &&
+                (allowedIncludes == null || allowedIncludes.contains(parameterName)
+                || allowedIncludes.contains(simplifiedParameterName)) &&
+                searchParam != null) {
             searchParameters.add("_include", parameterName);
         } else {
             if (LOG.isLoggable(Level.FINE)) {
                 LOG.fine("Filtering out searchParameter because it is not supported by the server config: " + parameterName);
             }
         }
     }
+
+    /**
+     * Read additional associated resources for the "extra" types (like location)
+     * supported by this server configuration.  Only LITERAL_RELATIVE references are
+     * resolved as LITERAL_ABSOLUTE references that match the server's hostname are
+     * normalized prior to this method.
```

Comment:
I think that technically they are normalized within this method, not prior to it
```suggestion
     * supported by this server configuration.  Only references of type LITERAL_RELATIVE (including
     * absolute references that match the server's hostname) are resolved by the implementation.
```

Reply:
Done